dev-docs/bidders/bluebillywig.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: bluebillywig 
+title: Blue Billywig
+description: Prebid Blue Billywig Bidder Adaptor
+biddercode: bluebillywig
+hide: true
+media_types: video
+gdpr_supported: true
+schain_supported: true
+coppa_supported: true
+usp_supported: true
+userIds: britepoolId, criteo, digitrust, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId
```

Comment:
Looks like you forgot to include support for unifiedId

```suggestion
userIds: britepoolId, criteo, digitrust, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId
```

Reply:
Oops, you're right, cheers!